main.js

```diff
@@ -1,8 +1,10 @@
-const oOverlay = require('./src/js/overlay');
+import oOverlay from './src/js/overlay';
+
 const constructAll = function() {
 	oOverlay.init();
 	document.removeEventListener('o.DOMContentLoaded', constructAll);
 };
 
 document.addEventListener('o.DOMContentLoaded', constructAll);
-module.exports = oOverlay;
+
+export default oOverlay;
```

Comment:
Been doing some typescript recently and there is an aversion to `export default` in favour of named exports. Is Origami going with `export default` for now? Read this article on how it could be harmful https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html

Reply:
Some components already export `default` but I hadn't really thought about it -- thanks for the link! 

Many of the points raised in the article don't apply to Origami components, I think using default makes sense as we rarely export more than one thing and renaming would be a breaking anyway. But the dynamic imports point is compelling. I'll discuss with the team tomorrow 👍 It might be we want to formalise one way or another in the spec.